### PR TITLE
Change A-left right to C-left right in insert

### PR DESCRIPTION
**Pull request by pickfire**
Currently A-left move one word left and the behavior will be more
consistent for people coming GUI world if the key was changed to control
given that both browsers and editors like vscode uses C-left right by
default to move word rather than alt.

I wanted to change A-backspace to C-backspace as well but I don't seemed to be able to do that. I saw this when looking at https://github.com/helix-editor/helix/issues/2131

cc @QiBaobin since he did the initial version but not sure why he did it that way, maybe consistency with alt-backspace?

### CODE REVIEW

**Review by QiBaobin — APPROVED** [review]

**Comment by EpocSquadron** [comment]
> I wanted to change A-backspace to C-backspace as well but I don't seemed to be able to do that. I saw this when looking at #2131

`Ctrl-backspace` is an ambiguous escape in most terminals, unless you have [CSIu](http://www.leonerd.org.uk/hacks/fixterms/)  support in the terminal emulator.

**Comment by pickfire** [comment]
Let's wait for a reviewer with write access to approve.

**Pull request merged**
